modules/core/src/matrix_wrap.cpp

```diff
@@ -981,21 +981,21 @@ bool _InputArray::isSubmatrix(int i) const
     if( k == STD_VECTOR_MAT )
     {
         const std::vector<Mat>& vv = *(const std::vector<Mat>*)obj;
-        CV_Assert((size_t)i < vv.size());
+        CV_Assert(i >= 0 && (size_t)i < vv.size());
         return vv[i].isSubmatrix();
     }
 
     if( k == STD_ARRAY_MAT )
     {
         const Mat* vv = (const Mat*)obj;
-        CV_Assert(i < sz.height);
+        CV_Assert(i > 0 && i < sz.height);
```

Comment:
Why >= somewhere and > here?

Reply:
The same assertion is used also in `_InputArray::isContinuous` for `STD_ARRAY_MAT` case above. I am not sure whether `>` is intentional for `STD_ARRAY_MAT` case or it is a typo. Maybe `>=` should be used in both places?